README.md

```diff
@@ -28,9 +28,31 @@ If you have questions or need help, please check out our documentation for a [li
 
 **Note: "All changes must be made in the `/src` folder and NOT in the `/docs` folder. Html files in the ./docs folder are auto generated "**
 
+## Code of conduct
+
+We at conda-forge adhere to the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct).
+
+Be kind to others. Do not insult or put down others. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for conda-forge.
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate.
+
+conda-forge is dedicated to providing a harassment-free community for everyone, regardless of gender, sexual orientation, gender identity and expression, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of community members in any form.
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+## Reporting Guidelines
+
+If you believe someone is violating the code of conduct, please report this in a timely manner. Code of conduct violations reduce the value of the community for everyone. The team at conda-forge takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.
+
+All reports will be kept confidential. Please have a look at the [Reporting Guidelines](https://numfocus.org/code-of-conduct#reporting-guidelines).
+
+## Enforcement: What Happens After a Report is Filed?
+
+conda-forge's team and/or our event staff will try to ensure your safety and help with any immediate needs, particularly at an in-person event. Once we have received the report through the relevant authorities, conda-forge will make every effort to acknowledge the receipt and take action. (and we’ll aim for much more quickly than that). Have a look at the process of [What Happens After a Report is Filed?](https://numfocus.org/code-of-conduct#enforcement).
```

Comment:
There is no need for the period after "action" since the words in the parenthesis are also part of the sentence.

Reply:
Fixed!